node/src/accountant/db_access_objects/receivable_dao.rs

```diff
@@ -364,42 +394,6 @@ impl ReceivableDaoReal {
             feeder.limit_clause
         )
     }
-
-    fn more_money_received_pretty_error_log(
```

Comment:
That's the point of the main change here. We used to roll back all the db operations if we ran into an error half the way. So this log was supposed to show the user all the transactions that we were about to write persistently even though we eventually had to give up. 

Now we don't need this big machinery because we agreed with Dan on panicking in case of any error different form the error caused by the transaction from an unknown address.